packages/tables/resources/js/components/table.js

```diff
@@ -2,6 +2,8 @@ export default function table() {
     return {
         collapsedGroups: [],
 
+        controller: null,
```

Comment:
Can you please rename the property to something like `checkboxClickController`, if that makes sense in this context?

Reply:
Yes, I've changed it!